independent-publisher-connectors/Fuzzy Search/apiProperties.json

```diff
@@ -0,0 +1,11 @@
+{
+   "properties": {
+     "connectionParameters": {},
+     "iconBrandColor": "#da3b01",
+     "capabilities": [
+       "actions"
+     ],
+     "publisher": "Mitanshu Garg",
+     "stackOwner": "Mitanshu Garg"
```

Comment:
@happyume stackOnwer should be the API Owner. 
"stackOwner": "Name of company or organization that connector is connecting to."